File: Javascript/Arrays/Eliminacion_final.js
/*
Eliminacion en Javascript, Eliminacion del ultimo elemento de un array.
@Autor: Daniel Alejandro Alvarado Velázquez
@grupo: #2-02
@Fecha: 03/09/2026
*/

//Declaramos al arreglo
const input_arr = [11, 21, 31, 41, 51, 61];

console.log("Antes de la eliminacion, el array es: ");
for (let i = 0; i < input_arr.length; i++){
    console.log(input_arr[i]);
}

input_arr.pop()     //remueve el ultimo elemento del arreglo

console.log("Despues de la eliminacion, el array es: ");
for (let i = 0; i < input_arr.length; i++){
    console.log(input_arr[i]);
}

File: C#/Arrays/Eliminacion_final.cs
/*
Eliminacion en Java, Eliminacion del ultimo elemento de un array.
@Autor: Daniel Alejandro Alvarado Velázquez
@grupo: #2-02
@Fecha: 04/09/2026
*/

using System;
using System.Linq;

namespace Eliminacion_final
{
    class Program
    {
        static void Main(String[] args)
        {
            int [] input_arr = {1, 11, 21, 31, 41, 51};
            int tam = input_arr.Length;

            Console.WriteLine("Antes de la eliminacion, el array es: ");
            foreach (int elemento in input_arr){
            Console.WriteLine(elemento);
            }

            input_arr = input_arr.Where((n,i) => i != tam -1).ToArray();        //arreglo = arreglo.where((n, i = indice), => indice != indice del elemento a borrar).toArray()
            //se modifica el arreglo 

            Console.WriteLine("Despues de la eliminacion, el array es: ");
            foreach (int elemento in input_arr){
            Console.WriteLine(elemento);
            }

        }
    }
}